### DISCUSSION TITLE
How to search for a folder?

### DISCUSSION THREAD

**Discussion by ghost** [nicotine-plus/nicotine-plus#2262] [Q&A] [answered]
Let's say I wanted to search for a `Jean-Claude Van Damme movie collection`. Since the search finds words in the files I would need to get the name of one of the movies and search for it. Then what? Should I browse the shares of each user that has that file to see if it is part of a collection?

It would be nice if I could select searching for a folder and write `Van Damme` and find only folders that include that name. Then filter the search by size and number of files. But the protocol probably doesn't allow it so I'm not even gonna ask.

Is there a better way of searching for a folder?

**Comment by mathiascode** [comment, marked as answer]
When you search for music collections, the easiest way is to search for the artist name, since albums are generally grouped by artist.

The issue with movies is that the names of the actors are rarely present in the file path. Movies are generally either ungrouped, or grouped by year or genre. You're probably better off looking up relevant movies elsewhere, searching for the movie names on Soulseek, and filtering by size or file extension. It's somewhat tedious, but Soulseek was only designed with music in mind.
